src/gz/gz_col_view.c

```diff
@@ -1038,3 +1051,77 @@ void gz_hit_view(void)
     gz.hit_view_state = HITVIEW_INACTIVE;
   }
 }
+
+static void do_path_list(Gfx **p_gfx_p, Gfx **p_gfx_d,
+                           z64_path_t *path_list)
+{
+  const float path_marker_radius = 18.0f;
+
+  z64_path_t *cur_path = path_list;
+
+  if (cur_path != NULL && (settings->bits.path_view_points || settings->bits.path_view_lines)) {
+    while (((cur_path->points << 4) >> 28) == Z64_SEG_SCENE && cur_path->numpoints != 0) {
+      z64_xyz_t *points = (z64_xyz_t*)((z64_stab.seg[Z64_SEG_SCENE]+0x80000000) + (cur_path->points & 0x00FFFFFF));
+      if (settings->bits.path_view_lines) {
+        load_l3dex2(p_gfx_p);
+        init_line_gfx(p_gfx_p, p_gfx_d, settings->bits.path_view_xlu);
+        for (int i = 0; i < cur_path->numpoints; ++i) {
+          if (i + 1 != cur_path->numpoints) {
+            draw_line(p_gfx_p, p_gfx_d, &points[i], &points[i+1]);
+          }
+        }
+        zu_set_lighting_ext(p_gfx_p, p_gfx_d);
+        unload_l3dex2(p_gfx_p);
+      }
+      if (settings->bits.path_view_points) {
+        for (int i = 0; i < cur_path->numpoints; ++i) {
+          draw_ico_sphere(p_gfx_p, p_gfx_d, points[i].x, points[i].y, points[i].z, path_marker_radius);
```

Comment:
I'm seeing some odd behavior with these spheres because the display list hasn't been properly initialized, `init_poly_gfx` and `gDPSetPrimColor` should be used at some point before this.